Dom.zavd 1/Salon.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+public class Salon
+{
+    private int carCount;
+    public Salon()
+    {
+        carCount = 0;

```

Comment:
You can set this value on field level without using constuctor, like `private int carCount = 0;`